databases/backends/postgres.py

```diff
@@ -128,6 +128,9 @@ def __iter__(self) -> typing.Iterator:
     def __len__(self) -> int:
         return len(self._row)
 
+    def __str__(self) -> str:
+        return f"""{{{", ".join(f"'{key}': {value!r}" for key, value in self.items())}}}"""
```

Comment:
Python counts references, so the dict will not be garbage collected. It will disappear immediately after exiting the visibility scope. Garbage collection does a different thing.

OK, then I have this suggestion:

```suggestion
    def __repr__(self) -> str:
        return f"Record({repr(self._row)}, {repr(self._result_columns)}, {repr(self._dialect)})"
```

This way the IDE is happy (it will use `__repr__` if `__str__` does not exist), and we leverage the Cython implementation of `asyncpg.Record.__repr__` which is potentially faster.